web-console/developer/topology/topology.md

```diff
@@ -3,5 +3,276 @@ The Topology page is a primary nav item in the OpenShift Developer perspective.
 
 Design documentation for the Topology section can be found [here](https://docs.google.com/document/d/1mIld9b_mC5sGZEvHHfwflQpFZVcrrwG029egE3550kg/edit#heading=h.k3mqw09n3tkg)
 
+## Topology
+The Topology view would be used in OpenShift Dev-Console to show semantic aggregated metadata about related pods, deployments, builds of application components in a namespace.
+
+![Topology View](img/Topology_View.png)
+
+
+### No Project in Context
+When no project is selected, Topology View will prompt users to select from a given list of projects.
+
+![No project in context](img/no_project_exists.png)
+
+
+#### No Workload Exists
+When no workload exists, the topology view redirects users to the add flows to start creating new workloads to be shown on the topology canvas.
+![Topology View](img/no_project_exists.png)
+
+### Application selector
+Users could choose to select either one of the applications or all the applications from the application selector.
+
+## Topology View
+
+
+### 1. Navigation item:
```

Comment:
is something missing here?  you have a section called Navigation item, but there's nothing associated with it.